### PR TITLE
Search bar logic

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: src/helpers/filters.ts [hunk 1]
```diff
@@ -64,6 +64,26 @@ const fileterByToAmount = (transactions: Transaction[], toAmount: number) => {
...
 };
 
+export const filterBySearch = (transactions: Transaction[], search: string) => {
+  return transactions.filter(transaction => {
+    const inputs: string[] = [];
+    const outputs: string[] = [];
+    transaction.inputs.filter(input => {
+      inputs.push(...input.addresses);
+    });
+    transaction.outputs.filter(output => {
```
Comment (bklukaczewski) [inline src/helpers/filters.ts:74]: `filter` shouldn't be used for that I think

have you considered using Array.reduce and without `Array.push` 
or `Array.forEach` with `Array.push`?

Comment (bklukaczewski) [reply src/helpers/filters.ts:74]: although Array.reduce would be the best

File: src/screens/Dashboard/TransactionList.tsx [hunk 2]
```diff
@@ -107,6 +111,7 @@ const styles = StyleSheet.create({
   noTransactionsContainer: {
     alignItems: 'center',
   },
+  footer: { height: 500 },
```
Comment (bklukaczewski) [inline src/screens/Dashboard/TransactionList.tsx:114]: This looks cheesy

Comment (Kamill90) [reply src/screens/Dashboard/TransactionList.tsx:114]: It moves a scroll view to the top when you got only one transaction after search. Should it be done in a different way?

**Pull request merged**